src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactory.java

```diff
@@ -1875,6 +1880,55 @@ private GencodeFuncotation createIgrFuncotation(final VariantContext variant,
         return funcotationBuilder.build();
     }
 
+    /**
+     * Creates a {@link GencodeFuncotation}s based on a given spanning deletion {@link Allele}.
+     *
+     * Reports reference bases as if they are on the {@link Strand#POSITIVE} strand.
+     * @param variant The {@link VariantContext} associated with this annotation.
+     * @param annotationTranscript The transcript ID to use for populating this funcotation.
+     * @param reference The {@link ReferenceContext} in which the given {@link Allele}s appear.
+     * @return An IGR funcotation for the given allele.
+     */
+    private GencodeFuncotation createFuncotationForSpanningDeletion(final VariantContext variant,
+                                                                    final String annotationTranscript,
+                                                    final ReferenceContext reference){
+
+        final GencodeFuncotationBuilder funcotationBuilder = new GencodeFuncotationBuilder();
+
+        // Get GC Content:
+        funcotationBuilder.setGcContent( calculateGcContent( variant.getReference(), Allele.SPAN_DEL, reference, gcContentWindowSizeBases ) );
+
+        funcotationBuilder.setVariantClassification( GencodeFuncotation.VariantClassification.COULD_NOT_DETERMINE )
+                .setRefAllele( variant.getReference() )
+                .setStrand(Strand.POSITIVE)
+                .setTumorSeqAllele2( Allele.SPAN_DEL_STRING )
+                .setStart(variant.getStart())
+                .setEnd(variant.getEnd())
+                .setVariantType(getVariantType(variant.getReference(), Allele.SPAN_DEL))
```

Comment:
`getVariantType` right now doesn't know about spanning deletions.  Can you add a case to that method that will correctly identify this as a deletion?

Reply:
I created a variant type of NA.  This variant spans a deletion, but is not a deletion itself.

Also, adding test to MAF rendering to make sure spanking deletions do not cause trouble.

Reply:
Done